test/test_sync.py

```diff
@@ -383,3 +383,80 @@ def test_multiple_tables(mc):
     sql = f"SELECT count(*) FROM {config.references[1].table} WHERE {config.references[1].driver_path_column}='{copied_file}'"
     gpkg_cur.execute(sql)
     assert gpkg_cur.fetchone()[0] == 1
+
+
+def test_sync_without_references(mc: MerginClient):
```

Comment:
it seems to me like 3x the same workflow with different start (config) conditions - I suggest to refactor this using pytest [parametrize](https://docs.pytest.org/en/7.1.x/how-to/parametrize.html)